[WNMGDS-535] Update intro page for doc site (#9)
CMSgov/hcgov-design-system: docs/src/pages/index.md

---
title: Introduction
weight: -100
---

> The [HealthCare.gov](https://www.healthcare.gov/) design system is a set of open-source design and front-end development resources for creating usable, accessible (e.g., WCAG/Section 508 compliant), responsive, and consistent websites and web-based solutions.

Reviewer: ```suggestion
The Healthcare.gov design system is a set of open-source design and front-end development resources for creating usable, accessible (e.g., WCAG/Section 508 compliant), responsive, and consistent Healthcare.gov websites.
```

> The Healthcare.gov design system is built on the CMS Design System and extends it to support additional CSS and React components.

Reviewer: ```suggestion
The Healthcare.gov Design System is built on the CMS Design System and extends it to support additional CSS and React components.
```

## Getting started

### Developers

> Developers and engineers are encouraged to use React which allows for more complex, interactive components, and many of our React components have built-in accessibility optimizations.

Reviewer: Hm, I feel like this info is duplicated from the [`Using components` page](https://design.cms.gov/startup/components/). Maybe we can just keep it simple and just have two bullet points like on [this page](https://design.cms.gov/startup/installation/)?

Reviewer: ### Developers

* **Recommended** [Install the `@cmsgov/design-system` NPM package](https://design.cms.gov/startup/installation/#install-using-npm)

* [Download the design system zip](https://design.cms.gov/startup/installation/#download-zip)
```

Author: I think from Kristen's perspective she was interested in having a bit more of the why on this page 
See the comments in the confluence page here https://confluence.cms.gov/display/HCDSG/Introduction+Page 

I'm ok with simplifying though

Reviewer: Ok, I think this is fine we can always iterate more in the future. Based on the discussion from the confluence theres a lot more to figure out in general, the audience, what info we want, etc.

Reviewer: One note though, the download the design system zip link should definitely go to our install page, this would address [Melissa's feedback here](https://confluence.cms.gov/display/HCDSG/Introduction+Page?focusedCommentId=382049021#comment-382049021). Right now we just have a direct download link and no instructions

Author: Thanks @bernardwang updated the NPM and download links to go to those sections of the installation page 👍

* **Recommended option:** Install the `@cmsgov/ds-healthcare-gov` Node package manager (npm) package if your project uses npm for package management.

    <a href="{{root}}/startup/installation/" class="ds-c-button">Install with npm</a>

> * If you're unfamiliar with npm and package management, [download the code as a zip file](https://github.com/CMSgov/design-system/releases/latest). 

Reviewer 2: ```suggestion
* If you're unfamiliar with npm and package management, [download the code as a zip file](https://github.com/CMSgov/hcgov-design-system/releases/latest). 
```

### Designers

> Design teams using are currently using Figma for creating UIs that follow HealthCare.gov's Design System but will be transitioning to using Sketch in the future. 

Reviewer 2: ```suggestion
Design teams are currently using Figma for creating UIs that follow HealthCare.gov's Design System but will be transitioning to using Sketch in the future. 
```

> Design teams are currently using Figma for creating UIs that follow HealthCare.gov's Design System but will be transitioning to using Sketch in the future. 

Reviewer: Is it worth linking out to the Figma here if its going to be transitioned away? and you need access too i thought. Maybe we can just have something similar to the Mgov homepage where we ask them to reach out to the team?

Author: Since there is no timeline for that transition and right now no resource listed for designer I feel it's OK to list Figma until we have something else - but agree we should add a way for folks to get access.

---
title: Introduction
weight: -100
---

> The [HealthCare.gov](https://www.healthcare.gov/) design system is a set of open-source design and front-end development resources for creating usable, accessible (e.g., WCAG/Section 508 compliant), responsive, and consistent websites and web-based solutions.

Reviewer: Hm, this content reads more like a README, and is pretty detailed. I was personally thinking something more concise and public facing, similar to the screenshot Barb shared [here](https://adhoc.slack.com/archives/C645GLLNN/p1605041312428900?thread_ts=1605034392.421300&cid=C645GLLNN). 

What do you think of that? Sorry, I know you already wrote out all this content!

Reviewer: Maybe we could ask designers or Kristen on HC.gov to help with this content?

Author: I can get feedback from some more designers

The Healthcare.gov design system is built on the CMS Design System and extends it to support additional CSS and React components.

> The design system is currently being applied to some pages on HealthCare.gov and project teams are actively working to apply the design system consistently across all of HealthCare.gov. 

Reviewer: I like this section, but maybe we should put it last

Reviewer: I'm also curious to what Kristen thinks. There are product using the design system that arent public facing or have these easily accessible links. I also think that we want other products to use the design system even if they arent using it as a dependency or with React.

Author: Kristen and Mel helped craft this content in a confluence doc prior to putting in this PR - I could see that content moving to the bottom though

Reviewer 3: I'm all for putting the list of applications at the bottom, good call. Please make sure to point out these are _a few_, not all of the applications currently using HC.gov DS.

* [Find Local Help](https://localhelp.healthcare.gov/#/)
* [Tax Tools](https://www.healthcare.gov/tax-tool/#/)
* [Window Shop](https://healthcare.gov/see-plans)

The design system is open-source, which means it's freely available to use by anyone!

## Getting started

### Developers

Developers and engineers are encouraged to use React which allows for more complex, interactive components, and many of our React components have built-in accessibility optimizations.

* **Recommended option:** Install the `@cmsgov/ds-healthcare-gov` Node package manager (npm) package if your project uses npm for package management.

    <a href="{{root}}/startup/installation/" class="ds-c-button">install with npm</a>

* If you're unfamiliar with npm and package management, [download the code as a zip file](https://github.com/CMSgov/design-system/releases/latest). 

### Designers

Design teams at CMS typically uses Sketch as the preferred user interface (UI) design tool. Take advantage of these resources when you're creating UIs that follow HealthCare.gov's Design System.

> * **Recommended option:** Install the HealthCare.gov Sketch library so we'll automatically receives updates when new versions of the Sketch Library are released by the HealthCare.gov team.

Reviewer 2: ```suggestion
* **Recommended option:** Install the HealthCare.gov Sketch library so you'll automatically receive updates when new versions of the Sketch Library are released by the HealthCare.gov team.
```
